electron/legendary/games.ts

```diff
@@ -234,24 +256,32 @@ Categories=Game;
     const enabledInStartMenu = GlobalConfig.get().config.enableDesktopShortcutsOnStartMenu
 
     if (enabledInDesktop || enabledInDesktop === undefined) {
-      writeFile(desktopFolder, shortcut, (err) => {
-        if(err)
-        {
-          logError(`${err.name}: ${err.message}`)
-        }
-        logError("Couldn't save shortcut to " + desktopFolder)
+      // spawn('echo', [shortcut, '>', ])
+      writeFile(desktopFolder, shortcut, () => {
+        logInfo('Shortcut saved on ' + desktopFolder)
       })
     }
     if (enabledInStartMenu || enabledInStartMenu === undefined) {
-      writeFile('/usr/share/applications', shortcut, (err) => {
-        if(err)
-        {
-          logError(`${err.name}: ${err.message}`)
-        }
-        logError("Couldn't save shortcut to /usr/share/applications")
+      writeFile(applicationsFolder, shortcut, () => {
```

Comment:
you need to handle err

Reply:
There is no err on writeFile. Handle callback has no arguments.
Even when everything is ok, the error was showing. So I did my research and writefile returns nothing on the callback.

Reply:
I mean, it has, but I guess not from graceful-fs, because it was returning nothing from the callback.
Anyway, lets try this on next Rc and check if someone will have any error creating those files.
I tested here with several games, adding, removing, installing and uninstalling, and its working in all cases.